tools/sidekick/config.json

```diff
@@ -32,6 +32,31 @@
       "excludePaths": [ "/**" ],
       "includePaths": [ "**/:x**" ]
     },
+    {
+      "containerId": "tools",
+      "title": "Send to CaaS",
+      "id": "sendtocaas",
+      "environments": [
+        "dev",
+        "preview",
+        "live",
+        "prod"
+      ],
+      "event": "send-to-caas",
+      "excludePaths": [
+        "https://milo.adobe.com/tools/caas**",
+        "*.json"
+      ]
+    },
+    {
+      "containerId": "tools",
```

Comment:
do we need the caas configurator here?
it does not actually make sense for our authors since we have custom component to render the caas cards

Reply:
I think that without this configuration, we can't have 'Send to CaaS' option on DME sharepoint

Reply:
Ah sorry, I thought you were referring to 'Send to CaaS' part. Updated in new commit: https://github.com/adobecom/dme-partners/pull/7/commits/9aa720c9513ad3825c29b0e970530d12457b1686